tests/configs/gpii.tests.acceptance.localInstall.config.json

```diff
@@ -9,5 +9,5 @@
             }
         }
     },
-    "mergeConfigs": "%universal/gpii/configs/gpii.config.development.all.local.json"
+    "mergeConfigs": "%universal/gpii/configs/gpii.config.all.development.dr.production.json"
```

Comment:
@kaspermarkus I need clarification on what "this one" is. Which two config files are identical?  Which one are you suggesting should be deleted?  I can find no file with the name you gave above:"gpii.tests.acceptance.dynamicDeviceReporter.config.json".